Common/src/CConfig.cpp

```diff
@@ -5175,6 +5183,35 @@ void CConfig::SetPostprocessing(SU2_COMPONENT val_software, unsigned short val_i
       nTurbVar = 2; break;
   }
 
+  /*--- Postprocess SST_OPTIONS into structure. ---*/
+  //TODO this wont work.
+  string sst_string = "SST";
+  struct SST_Struct {
```

Comment:
Im not entirely sure how create this structure.  But I imagine I can pass this structure into the turb solver like how we currently pass the sustaining_terms bool?